recipe/meta.yaml

```diff
@@ -10,21 +10,34 @@ source:
   sha256: 768c50ebf5823f8cde81a9e38ffff115c8f5a5d031a37520d0024e7b9c6cd22e
 
 build:
-  skip: true  # [win]
-  number: 0
+  number: 1
+  skip: true  # [win and py27]
+  
+  features:
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
```

Comment:
Maybe make this match what's in gflags-feedstock? https://github.com/conda-forge/gflags-feedstock/blob/master/recipe/meta.yaml#L20